game/plugins/src/main/kotlin/gg/rsmod/plugins/content/areas/spawns/spawns_10057.plugin.kts

```diff
@@ -0,0 +1,7 @@
+/**
+ * WILDERNESS NPC & ITEM SPAWNS
+ * @author Harley Gilpin <https://github.com/HarleyGilpin>
+ */
+spawn_npc(npc = Npcs.KOLODION, x = 2541, z = 4716, walkRadius = 3, direction = Direction.WEST)
+spawn_npc(npc = Npcs.GUNDAI, x = 2534, z = 4713, walkRadius = 3, direction = Direction.WEST)
+spawn_npc(npc = Npcs.LUNDAIL, x = 2535, z = 4719, walkRadius = 0, direction = Direction.SOUTH)
```

Comment:
NPCs that don't walk do not need their walkRadius set to 0, simply not having it there will make them stationary. To finish the review, please fix any issues by removing `walkRadius = 0` :)